src/genie/libs/parser/iosxe/tests/ShowWlanAll/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,340 @@
+                    "wpa_ssn_ie": "Disabled"
+                },
+                "wifi_protected_access_wpa2": {
+                    "wpa2_rsn_ie": "Enabled",
+                    "wpa2_attributes": {
+                        "mpsk": "Disabled",
+                        "aes": "Enabled",
+                        "ccmp256": "Disabled",
+                        "gcmp128": "Disabled",
+                        "gcmp256": "Disabled",
+                        "randomized_gtk": "Disabled"
+                    }
+                },
+                "wifi_protected_access_wpa3": {
+                    "wpa3_ie": "Disabled"
+                },
+                "auth_key_mgmt": {
+                    "dot1x": "Enabled",
+                    "psk": "Disabled",
+                    "cckm": "Disabled",
+                    "ft_dot1x": "Enabled",
+                    "ft_psk": "Disabled",
+                    "dot1x_sha256": "Disabled",
+                    "psk_sha256": "Disabled",
+                    "sae": "Disabled",
+                    "owe": "Disabled",
+                    "suiteb_1x": "Disabled",
+                    "suiteb192_1x": "Disabled"
+                },
+                "owe_transition_mode": "Disabled",
+                "osen": "Disabled",
+                "ft_support": "Enabled",
+                "ft_support_attributes": {
+                    "ft_reassociation_timer_secs": 20,
+                    "ft_over_the_ds_mode": "Disabled"
+                },
+                "pmf_support": "Disabled",
+                "pmf_support_attributes": {
+                    "pmf_association_comeback_timeout_secs": 1,
+                    "pmf_sa_query_time_msecs": 200
+                },
+                "web_based_authenticaion": "Disabled",
+                "conditional_web_redirect": "Disabled",
+                "splash_page_web_redirect": "Disabled",
+                "webauth_on_mac_filter_failure": "Disabled",
+                "webauth_authentication_list_name": "Disabled",
+                "webauth_authorization_list_name": "Disabled",
+                "webauth_parameter_map": "Disabled"
+            },
+            "band_select": "Disabled",
+            "load_balancing": "Disabled",
+            "multicast_buffer": "Disabled",
+            "ip_source_guard": "Disabled",
+            "assisted_roaming": {
+                "neighbbor_list": "Enabled",
+                "prediction_list": "Disabled",
+                "dual_band_support": "Disabled"
+            },
+            "ieee_dot11v_parameters": {
+                "directed_multicast_service": "Enabled",
+                "bss_max_idle": "Enabled",
+                "bss_max_idle_attributes": {
+                    "protected_mode": "Disabled"
+                },
+                "traffic_filtering_servce": "Disabled",
+                "bss_transition": "Enabled",
+                "bss_transition_attributes": {
+                    "disassocitation_imminent": "Disabled"
+                },
```

Comment:
same idea with other comment.
```suggestion
                "bss_transition": {
                    "bss_transition_status": "Enabled",
                    "disassociation_imminent": {
                    "disassocitation_imminent_status": "Disabled"
                    "optimised_roaming_timer": 40,
                    "timer": 200,
                },
```
below lines were missed?
```
            Optimised Roaming Timer            : 40
            Timer                              : 200

Reply:
Updated for both schema and missed lines.